scripts/prepare-preview-builds.jq

```diff
@@ -8,14 +8,21 @@
 # This is just for testing, it doesn't need to strictly follow SemVer.
 .version |= split("-")[0] + "-preview.\($hash)" |
 
+# Update the build script in the root package.json to use the preview scope
+if .scripts and .scripts.build then
+  .scripts.build |= gsub("@metamask/"; "\($npm_scope)/")
+else
+  .
+end |
```

Comment:
Updates the react native packages in the build script to use @metamask-previews 
- https://github.com/MetaMask/metamask-design-system/blob/main/package.json#L16